pipelines/controller/README.md

```diff
@@ -1,2 +1,80 @@
-Please see
-[this Wiki page](https://github.com/google/fhir-data-pipes/wiki/Try-out-the-FHIR-Pipelines-Controller).
+# FHIR Pipelines Controller
+
+The FHIR Pipelines Controller helps you schedule and manage the transformation
+of data from a [HAPI FHIR server](https://hapifhir.io/) to a collection of
+Apache Parquet files. It uses the FHIR Data Pipes JDBC pipeline to run either
+full or incremental transformations to a Parquet data warehouse.
+
+The FHIR Pipelines Controller only works with HAPI FHIR servers using Postgres.
+You can see
+[an example of configuring a HAPI FHIR server to use Postgres here](https://github.com/hapifhir/hapi-fhir-jpaserver-starter#postgresql-configuration).
+
+## Usage
+
+### Setup
+
+Clone the
+[fhir-data-pipes GitHub repository](https://github.com/google/fhir-data-pipes),
+open a terminal window, and `cd` to the directory where you cloned it. Later
+terminal commands will assume your working directory is the repository root.
+
+Next, configure the FHIR Pipelines Controller. The FHIR Pipelines Controller
+relies on several configuration files to run. Edit them to match your
+environment and requirements.
+
+- <code>[pipelines/controller/config/application.yaml](https://github.com/google/fhir-data-pipes/blob/master/pipelines/controller/config/application.yaml)</code>
+  - The main settings for the FHIR Pipelines Controller. Edit the values to
+    match your HAPI FHIR server and use case.
+- <code>[pipelines/controller/config/hapi-postgres-config.json](https://github.com/google/fhir-data-pipes/blob/master/pipelines/controller/config/hapi-postgres-config.json)</code>
+  - JDBC settings. Edit the values to match the Postgres database of your HAPI
+    FHIR server.
+- <code>[pipelines/controller/config/flink-conf.yaml](https://github.com/google/fhir-data-pipes/blob/master/pipelines/controller/config/flink-conf.yaml)</code>
```

Comment:
I think for this to have an affect, we should also set this env. var: `export FLINK_CONF_DIR=[PATH]/config`. Please also clarify that this has to be set otherwise the updated value of `taskmanager.memory.network.max` is not picked up and the default will produce memory errors for large datasets (this is also needed to be adjusted if the machine does not have a lot of memory).